R/utilities-units.R

```diff
@@ -288,6 +288,7 @@ getUnitsForDimension <- function(dimension) {
   validateIsString(dimension)
   dimensionTask <- getDimensionTask()
   rClr::clrCall(dimensionTask, "AllAvailableUnitNamesFor", enc2utf8(dimension))
+  stop("errror")
```

Comment:
@IndrajeetPatil THis is on purpose just to ask you why the other code does not work